src/LanguageServer/ProtocolUnitTests/Diagnostics/PullDiagnosticTests.cs

```diff
@@ -56,18 +56,15 @@ public async Task TestDocumentDiagnosticsForOpenFilesWithFSAOff(bool useVSDiagno
         var markup = @"class A {";
         await using var testLspServer = await CreateTestWorkspaceWithDiagnosticsAsync(markup, mutatingLspWorkspace, BackgroundAnalysisScope.OpenFiles, useVSDiagnostics);
 
-        // Calling GetTextBuffer will effectively open the file.
-        testLspServer.TestWorkspace.Documents.Single().GetTextBuffer();
-
         var document = testLspServer.GetCurrentSolution().Projects.Single().Documents.Single();
 
         await OpenDocumentAsync(testLspServer, document);
 
         var results = await RunGetDocumentPullDiagnosticsAsync(
             testLspServer, document.GetURI(), useVSDiagnostics);
 
-        Assert.Equal("CS1513", results.Single().Diagnostics.Single().Code);
-        Assert.NotNull(results.Single().Diagnostics.Single().CodeDescription!.Href);
+        Assert.Equal("CS1513", results.Single().Diagnostics!.Single().Code);
```

Comment:
everything in this file is removal of the text buffer calls and nullable updates